sdk/src/transaction_context.rs

```diff
@@ -249,6 +254,18 @@ impl TransactionContext {
     pub fn get_instruction_trace(&self) -> &InstructionTrace {
         &self.instruction_trace
     }
+
+    /// Returns (in bytes) how much data can still be allocated
+    pub fn get_total_resize_remaining(&self) -> u64 {
```

Comment:
If used incorrectly, this method could lead to indeterminism due to how the `total_resize_limit` is set when the bank constructs the `TransactionContext`.

Reply:
This method is not accessible to BPF programs yet anyway. And for builtin programs we could mark it as do not use / only for testing.

Reply:
> And for builtin programs we could mark it as do not use / only for testing.

This would be great, can you please do that for now?

> The "limit" part of the AccountsDataMeter (and this move into TransactionContext) will be removed, so this fn will be removed

Ok, then why did we add this fn 2 days ago if we are about to remove it?